sentry-android-timber/src/main/java/io/sentry/android/timber/SentryTimberTree.kt

```diff
@@ -12,7 +12,7 @@ import timber.log.Timber
  * Sentry Timber tree which is responsible to capture events via Timber
  */
 @Suppress("TooManyFunctions") // we have to override all methods to be able to tweak logging
-class SentryTimberTree(
+internal class SentryTimberTree(
```

Comment:
I think we should keep this public, some people use it directly, e.g. https://github.com/natsumerinchan/FoxMagiskModuleManager/blob/253ccb91270b586f50fdb2010391148dbd44ad9e/app/src/main/java/com/fox2code/mmm/utils/TimberUtils.kt#L7